frontend/src/App.css

```diff
@@ -51,6 +51,11 @@
 	padding: 0;
 	margin: 0;
 }
+
+.dialog-modal:focus-visible {
+	outline-offset: 0.2rem;
```

Comment:
I code after dark because I'm a nerd and in my personal projects I set the base font-size to 10px, which makes working with rem a million times easier. But yes, I'll change this to 0.1875rem.